modules/dnn/test/test_layers_1d.cpp

```diff
@@ -245,4 +245,30 @@ INSTANTIATE_TEST_CASE_P(/*nothing*/, Layer_Elemwise_1d_Test, Combine(
 /*operation*/           Values("div", "prod", "max", "min", "sum")
 ));
 
+TEST(Layer_Reshape_Test, Accuracy)
```

Comment:
This test verifies only 0D case. This **MUST** be exposed in the test name or its parameters.